rest/route/project.go

```diff
@@ -1048,3 +1048,59 @@ func (h *projectParametersGetHandler) Run(ctx context.Context) gimlet.Responder
 
 	return gimlet.NewJSONResponse(res)
 }
+
+////////////////////////////////////////////////////////////////////////
+//
+// PUT /rest/v2/projects/value/{toReplace}/{replacement}
+
+type rotation struct {
```

Comment:
1. can we call this projectVarsPutInput or something instead? (or whatever matches the handler).
2. I think it would also make sense to have a dry_run option, so that users can see what projects _would_ be affected. (We should return a string list of the projects that are impacted I think). 
3. These variables need json tags: the user would expect to use snake case "to_replace" to indicate this, for example, and without json tags the user would need to do "toreplace" (because the default is the existing variable name in lowercase)